Cargo.toml

```diff
@@ -36,6 +36,8 @@ crate-type = ["lib", "staticlib"]
 
 [features]
 default = ["vendored-libbpf"]
+# Android platform support
+android = [ "bindgen", "vendored" ]
```

Comment:
I don't really understand why we need the feature. Isn't Android "just" a different platform/target? This should be handled transparently, in my opinion. Conversely, it makes little sense for anyone building the library to be able to select to build for Android when really that would require existence of a cross compilation toolchain in the general case? I.e., if all I have is a "standard" Linux, will enabling this feature result in Android compatible binaries out of the box?

Reply:
You're right, it doesn't have much practical use here. Initially, it was added to 'Android' because of fear of disrupting the build process